ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -86,6 +94,11 @@ sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" manifests/cluste
 sed -i "/publicZone/,+1d" manifests/cluster-dns-02-config.yml
 sed -i "/privateZone/,+1d" manifests/cluster-dns-02-config.yml
 
+if [ -v vnet_basename ] && [ -n "${vnet_basename}" ]; then
+  echo "Editing vnet NSG from the existing vnet"
+  installer_infraID=$(cat .openshift_install_state.json | jq -j '."*installconfig.ClusterID".InfraID')
+  sed -i "s/${installer_infraID}-nsg/${vnet_basename}-nsg/g" manifests/cloud-provider-config.yaml
+fi
```

Comment:
Set a nsg name variable here to use later when deleting.

Reply:
Done